cypress/integration/closing-a-week.spec.js

```diff
@@ -532,7 +532,6 @@ describe('Closing a week', () => {
       cy.get('.bc-open-weeks__period:nth-of-type(1)').within(() => {
         cy.get('.bc-open-weeks__week:nth-of-type(1)').within(() => {
           cy.get('header').within(() => {
-            cy.contains('h4', 'Operatives with no SMVs (0)')
```

Comment:
This is not really doing anything (not set up as an assertion), is that why it was removed?

Reply:
See https://circleci-tasks-prod.s3.us-east-1.amazonaws.com/storage/artifacts/72d9e718-b0ad-48ba-9181-cecce0b0736a/e2484baf-ac2c-4b77-877b-57923081b73b/0/cypress/screenshots/closing-a-week.spec.js/Closing%20a%20week%20--%20When%20logged%20in%20as%20a%20week%20manager%20--%20They%20can%20close%20the%20week%20%28failed%29.png?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=ASIAQVFQINEOIPOEPSNV%2F20230126%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20230126T151736Z&X-Amz-Expires=60&X-Amz-Security-Token=IQoJb3JpZ2luX2VjELj%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FwEaCXVzLWVhc3QtMSJHMEUCIA5Smm0H3DZCHKq0G%2BvxOuNZZjsR9pJhDkyz3XxYUfDAAiEAvEQFhQ9G6%2FY4NGd9mMwVmxkAZAksTMmVpIe5xXqVtaAqqwIIMBADGgwwNDU0NjY4MDY1NTYiDHCLHpAmOALW8d4I2iqIAkhhRJvlj%2ByjzKqucasQt%2BSGIztZaLeB8H1o9vIeswGLQ5a5uRjfyRSaa3Ko3lMLTV6qib3NChux%2BPQBJIz32pTRVkQJBLpS4gkXo00Y%2FFAmSSNovmVu5B%2FNFTGB2DXruQ6IKflR5Z9T%2FzqiQ0hERcC21deZBFlqd6RbozEGFMFoPavcP7TtB3ZCb%2FpTifjicFFKsErLQ4Jlwp7kqierChjiRl6q1xCbtLPRyxgZ4dq3DxRqvmZPGD7Igf3dAbiztuiMcKdKgQAvMMa647L%2BrTEL1by10N2wqg0bsYDotsGihOnSZIPAfxJJP7hGt7ipz5QGo0Oeo3B%2BYeAf5gWjPUWxxKPVDx5kuzCsrcqeBjqdAQmczG2zY%2BdNmTAewsXwHVZ1EbDGByxbo0twI9Iyp%2BibexQq30z42V2HNTumhwvBlTdoRVN1rwDPj1w0T367lgXc7ee88%2BiBsMuPm9cX627ngFqziQD%2BofyfoMwnYALzmHNQ2HSOZ1Y31ohT6kb0nvKaOiei2YqDZn6tDE69TRE7yI2s7VnlaurfUPuSC922fC%2Bz9TDPQwolYy4drec%3D&X-Amz-SignedHeaders=host&x-id=GetObject&X-Amz-Signature=bd98067775400d573c98f549b8983681837b28b7dd0b6b1e5b1a8b6a25cd803f

Reply:
I get this
![image](https://user-images.githubusercontent.com/70756861/214900318-8decf30f-6882-484b-999e-b82851b6f4ac.png)